int8 leftcamera
float64 leftangle
int8 rightcamera
float64 rightangle